lib/shrinkwrap-package.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var Package = require('./package');
+
+function ShrinkwrapPackage(name, versionSpecified, versionInstalled, parents) {
```

Comment:
Why not just change `Package` to include parents? Do we need a separate `ShrinkwrapPackage` object?